server/schedulers/hot_region.go

```diff
@@ -74,6 +74,12 @@ const (
 
 	minHotScheduleInterval = time.Second
 	maxHotScheduleInterval = 20 * time.Second
+
+	divisor                   = float64(1000) * 2
```

Comment:
do we need to config `divisor`?

Reply:
We can make it configurable if it is needed in the future.